nservicebus/messaging/delayed-delivery.md

```diff
@@ -13,6 +13,16 @@ The message doesn't have to be dispatched immediately after sending, it can be d
 
 NOTE: Only send operations can be deferred. Publish and reply operations cannot be deferred.
 
+## Handling current message later
+To handle the current message at a later time `bus.HandleCurrentMessageLater()` method can be used.  This was the initial way of *deferring* a message. This would create a copy of the message that has the same identifier, header and body, and then put the new message at the end of the queue. The message eventually will be pick up again once other messages in the queue are processed. 
```

Comment:
>  This was the initial way of *deferring* a message.

not sure about the value of this information. Should we instead point out that this method has been superseded by the defer operations?